### DISCUSSION TITLE
Vulkan Render stops at launch of a beatmap.

### DISCUSSION THREAD

**Discussion by msmcculley** [ppy/osu#22969] [Q&A]
[network.log](https://github.com/ppy/osu/files/11072494/network.log)
[performance.log](https://github.com/ppy/osu/files/11072495/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/11072496/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/11072497/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/11072498/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/11072499/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/11072500/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11072502/updater.log)
[database.log](https://github.com/ppy/osu/files/11072503/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11072504/legacy-ipc.log)

![](https://i.imgur.com/L4yYSB0.png)

https://user-images.githubusercontent.com/90729492/227794475-13d2ceef-6c16-445a-a4fe-c4ccc553c448.mp4

Pretty much the title: 

Everything else works fine but launching a beatmap, and any beatmap, causes it to freeze

This can be reproduced by switching to Vulkan, and going to any beatmap and attempt to play it.

To get osu! back to the way it was i need to completely close out of the game, there is no going back after this point and it doesnt seem to fix itself if the beatmap ends either. (This is a little different from other discussions because they are encountering a blackscreen and are not of the same GPU, but nonetheless, here we go).

**Comment by bdach** [comment]
https://github.com/ppy/osu/issues/22960
